src/plugins/legacy/vtkDataMesh/readers/vtkDataMeshReaderBase.h

```diff
@@ -0,0 +1,42 @@
+/*=========================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2013 - 2019. All rights reserved.
+ See LICENSE.txt for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+=========================================================================*/
+#pragma once
+
+#include "vtkDataMeshPluginExport.h"
+
+#include <dtkCoreSupport/dtkAbstractDataReader.h>
+
+class vtkMetaDataSet;
+
+class VTKDATAMESHPLUGIN_EXPORT vtkDataMeshReaderBase: public dtkAbstractDataReader
+{
+    Q_OBJECT
+
+public:
+    vtkDataMeshReaderBase();
+
+public slots:
+
```

Comment:
Be careful :
 "virtual bool canRead (const QStringList& paths);" is the old way to specify a function is heritage and already virtual.
new way :
"bool canRead (const QStringList& paths) override;"